qt_ui/widgets/QTopPanel.py

```diff
@@ -101,6 +103,8 @@ def setGame(self, game: Optional[Game]):
         self.budgetBox.setGame(game)
         self.factionsInfos.setGame(game)
 
+        self.passTurnButton.setEnabled(True)
```

Comment:
Should be `self.passTurnButton.setEnabled(game is not None)`, but otherwise change looks good. Thanks again!

(if you're wondering, I think the case where this is called with `None` is if a game fails to load for whatever reason)

Reply:
But that will always be True since at line 99 there is a check for game is None

Reply:
Whoops, yes, you're right :) Merging.